x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -47,7 +48,8 @@ export const FormContext: React.FC<Props> = ({
   onSuccess,
 }) => {
   const { connectors, loading: isLoadingConnectors } = useConnectors();
-  const { connector: configurationConnector } = useCaseConfigure();
+  const owner = useOwnerContext();
+  const { connector: configurationConnector } = useCaseConfigure(owner[0]);
```

Comment:
Same thing here about just calling the context in the calls. The same can probably be done for the postCase if it follows the other container architecture